packages/talos_messenger_core/src/services/inbound_service.rs

```diff
@@ -145,14 +145,14 @@ where
         loop {
             tokio::select! {
                 // 1. Consume message.
-                Ok(Some(msg)) = self.message_receiver.consume_message() => {
+                // Ok(Some(msg)) = self.message_receiver.consume_message() => {
+                reciever_result = self.message_receiver.consume_message() => {
 
-                    // 2. Add/update to suffix.
-                    match msg {
+                    match reciever_result {
                         // 2.1 For CM - Install messages on the version
-                        ChannelMessage::Candidate(candidate) => {
-
+                        Ok(Some(ChannelMessage::Candidate(candidate))) => {
                             let version = candidate.message.version;
+                            info!("Candidate version received is {version}");
```

Comment:
Is hits info important? Can it be debug instead of info?

Reply:
Aaah, it shouldn't be `info!` will update that to debug